cli/command/container/cp_test.go

```diff
@@ -178,13 +178,14 @@ func TestSplitCpArg(t *testing.T) {
 			expectedContainer: "container",
 		},
 	}
-	for _, testcase := range testcases {
-		t.Run(testcase.doc, func(t *testing.T) {
-			skip.If(t, testcase.os != "" && testcase.os != runtime.GOOS)
-
-			ctr, path := splitCpArg(testcase.path)
-			assert.Check(t, is.Equal(testcase.expectedContainer, ctr))
-			assert.Check(t, is.Equal(testcase.expectedPath, path))
+	for _, tc := range testcases {
+		tc := tc
+		t.Run(tc.doc, func(t *testing.T) {
+			skip.If(t, tc.os == "windows" && runtime.GOOS != "windows" || tc.os == "linux" && runtime.GOOS == "windows")
```

Comment:
nit: This could be a bit more readable with:
```suggestion
            windowsTestOnNonWindowsOs := tc.os == "windows" && runtime.GOOS != "windows"
            linuxTestOnWindows := tc.os == "linux" && runtime.GOOS == "windows"
			skip.If(t, windowsTestOnNonWindowsOs || linuxTestOnWindows)
```

Reply:
Ah, yeah, this one was a bit ugly. Maybe `unix` as value in the test, a `IsUnix()` utility; although I guess we could use `os.PathSeparator`;

```go
tc.os == "unix" && os.PathSeparator != "/" || tc.os == "windows" && os.PathSeparator != `\`
```

I may do some follow-ups to this one; I'll have a look at part of those 👍